engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/compile/nodecompilation/NodeCompiler.scala

```diff
@@ -149,9 +149,9 @@ class NodeCompiler(definitions: ProcessDefinition[ObjectWithMethodDef],
     NodeCompilationResult(expressionTypingInfo, None, newCtx, validParams)
   }
 
-  def compileFields(fields: List[graph.variable.Field], 
+  def compileFields(fields: List[graph.variable.Field],
                     ctx: ValidationContext,
-                    outputVar: Option[OutputVar])
+                    maybeOutputVar: Option[OutputVar])
```

Comment:
why maybeOutputVar? We already know that from signature?

Reply:
Because there is option?